### ISSUE TITLE
JS-style Comments Incorrectly Flagged as [style-lint] Unknown word (Css Syntax Error)

### ISSUE THREAD

**Issue by Undistraction** [styled-components/stylelint-processor-styled-components#199]
Using JS-style comments (which is [supported](https://github.com/styled-components/stylelint-processor-styled-components/pull/164)) inside a styled component's backticks results in the comment being flagged as an error:

## Example
```
const Layout = styled.div`
  // Comment
`
```
The first `/` will be flagged with:

> [style-lint] Unknown word (Css Syntax Error)

## Workaround

The only workaround is to add a disabling comment before the comments.

tylelint-processor-styled-components@1.3.1
stylelint@9.2.1

**Comment by Undistraction** [comment]
Sorry. There is already an issue #195
